HW_3/task_1.js

```diff
@@ -0,0 +1,42 @@
+/*
+1. Создайте цикл, который выведет в консоль цифры от 10 до 0
+2. Написать скрипт, который выведет 5 строк в консоль таким образом, чтобы в первой строчке выводилось :), во второй :):) и так далее
+  Пример в консоли:
+  :)
+  :):)
+  :):):)
+  :):):):)
+  :):):):):)
+
+  Для Оксаны и ее друзей: также реализовать циклом while
+  Рекоммендация: попробуйте метод .repeat()
+
+
+*/
+// by For
+for(i=0; i<=10; i++){
+    console.log(i)
+}

```

Comment:
Forgot to explicitly declare a variable to start of the loop
Pay attention to task description - `output from 10 to 0`